src/pages/who-we-are/_OurOffices.tsx

```diff
@@ -173,7 +173,7 @@ const OurOffices = () => {
             </StyledHeader>
 
             <Flex>
-                <Show.Desktop max_width="tabletL">
+                <Desktop max_width="tabletL">
```

Comment:
I don't think `Desktop` and `Mobile` uses `max_width` props. Please take a look at the `visibility.tsx` file.

Reply:
The Desktop component renders a DesktopLayer component which has a media query where max_width uses a breakpoint value. (Similarly with Mobile component , MobileLayer and min_width.)

Reply:
Yep but the prop should be `breakpoint` in that case. This is still working as the default of the `Desktop` component is `tabletL`.

Reply:
yes you are right, I will make the necessary changes and update.